packages/core/src/internal/utils/identity.ts

```diff
@@ -45,3 +45,7 @@ export function hasStringPropertyChangedAndNotNil(val: string | null | undefined
 export function getEnumValues(enumeration: any) {
   return Object.values(enumeration);
 }
+
+export function createId() {
+  return `_${Math.random().toString(36).substr(2, 9)}`;
```

Comment:
Not sure, I'll revisit this. There was something that was causing some issues for me but it may not be the case anymore. I think it was related to multiple unique ids on the same class maybe.

Reply:
fwiw, i'm cool with getting rid of unique-id. but we should either do it all now or open a github issue to do it. (it would be an internal refactor so not a breaking change)